src/js/pose_viewer/src/components/pose-viewer/pose-viewer.tsx

```diff
@@ -2,6 +2,7 @@
 import {Component, Element, Event, EventEmitter, h, Host, Method, Prop, State, Watch} from '@stencil/core';
 import type {PoseModel} from "pose-format/dist/types";
 import {Pose} from "pose-format";
+import {Buffer} from "buffer";
```

Comment:
i would prefer an `import type {Buffer}` so buffer does not transpile to javascript, as it is not available in all environments